x/authz/client/cli/query.go

```diff
@@ -91,3 +93,101 @@ $ %s query %s grants link1skjw.. link1skjwj.. %s
 	flags.AddPaginationFlagsToCmd(cmd, "authorizations")
```

Comment:
Please check why the query string is different.

cosmos-sdk, "grants"
https://github.com/cosmos/cosmos-sdk/blob/f2d94445c0f5f52cf5ed999b81048b575de94964/x/authz/client/cli/query.go#L93

But, lbm-sdk, "authorizations"

Reply:
changed it. thx!